Gemfile

```diff
@@ -2,3 +2,4 @@ source "http://rubygems.org"
 
 # Specify any dependencies in the gemspec
 gemspec
+# gem 'byebug', platforms: [:mri_20, :mri_21, :mri_22]
```

Comment:
I left this comment here as a reminder to people who want to add byebug back. It should be added in the Gemfile because it's not supported by all versions of Ruby, and you can't target platforms in the gemspec. I commented it out because all configurations of Travis CI balked at `:mri_22` except for the one that was actually _running_ MRI 2.2.